cwf/gateway/go.sum

```diff
@@ -103,6 +103,7 @@ github.com/dgraph-io/ristretto v0.0.2/go.mod h1:KPxhHT9ZxKefz+PCeOGsrHpl1qZ7i70d
 github.com/dgrijalva/jwt-go v3.2.0+incompatible/go.mod h1:E3ru+11k8xSBh+hMPgOLZmtrrCbhqsmaPHjLKYnJCaQ=
 github.com/dgryski/go-farm v0.0.0-20190423205320-6a90982ecee2/go.mod h1:SqUrOPUnsFjfmXRMNPybcSiG0BgUW2AuFH8PAnS2iTw=
 github.com/dgryski/go-sip13 v0.0.0-20181026042036-e10d5fee7954/go.mod h1:vAd38F8PWV+bWy6jNmig1y/TA+kYO4g3RSRF0IAv0no=
+github.com/dnephin/pflag v1.0.7/go.mod h1:uxE91IoWURlOiTUIA8Mq5ZZkAv3dPUfZNaT80Zm7OQE=
```

Comment:
Is this change intentional? (not a go expert here)

Reply:
Does our workflow force these `go.mod` and `go.sum` changes, or can they be avoided @christinewang5 ?

Reply:
Yeah it was auto-added, but with the revert here https://github.com/magma/magma/pull/12500, we probably don't even need this PR anymore, so closing for now.

Reply:
I closed #12500 in favor of this task :)